src/backend/marsha/core/tests/test_api_update_state.py

```diff
@@ -220,15 +220,20 @@ def test_api_update_state_timed_text_track(self):
         }
         signature = generate_hash("shared secret", json.dumps(data).encode("utf-8"))
         with mock.patch(
-            "marsha.websocket.utils.channel_layers_utils.dispatch_video_to_groups"
-        ) as mock_dispatch_video_to_groups:
+            "marsha.websocket.utils.channel_layers_utils.dispatch_video"
```

Comment:
```suggestion
            "marsha.websocket.utils.channel_layers_utils.dispatch_video"
```
Someone wise told me not to use the full path with mocking but to import in the file dispatch_video and then to patch it.

Reply:
![](https://media.giphy.com/media/dYTfJZ2dCQBhK/giphy-downsized.gif)